src/lib/Colors.json

```diff
@@ -12,19 +12,31 @@
     },
     "difficultyColor": {
         "EASY": {
-            "backgroundColor": "rgba(25, 186, 63, 0.3)",
+            "backgroundColor": {
```

Comment:
This is actually a bad way to store data. With the refactor this should be changed and solved with css